### PR TITLE
[PSMDB-122] percona-server-mongodb-enable-auth.sh doesn't work from binary tarball

### PR DESCRIPTION
**Pull request by EvgeniyPatlan**

### CODE REVIEW

File: percona-packaging/conf/percona-server-mongodb-enable-auth.sh [hunk 1]
```diff
@@ -8,28 +8,40 @@ else
...
-[ -z "${CONF}" ] && CONF=/etc/mongod.conf
 
-if [ "$1" == "-h" -o "$1" == "--help" ] ; then
-    echo "Usage: `basename $0` [-h] [-q]"
+usage() { 
+    echo "Usage: `basename $0` [-h] [-q] [-c]"
     echo
     echo "  This script is used to enable authentication in Percona Server for MongoDB automatically."
     echo "  User 'dba' with role 'root' will be created with autogenerated password and the service will be restarted."
     echo "  -q - quiet mode. Do not ask any questions and enable auyh automatically"
```
Comment (tplavcic) [inline percona-packaging/conf/percona-server-mongodb-enable-auth.sh]: "auyh" typo?

Comment (EvgeniyPatlan) [inline percona-packaging/conf/percona-server-mongodb-enable-auth.sh]: fixed

Comment (EvgeniyPatlan) [inline percona-packaging/conf/percona-server-mongodb-enable-auth.sh]: both issues are fixed.

**Comment by tplavcic** [comment]
LGTM

**Pull request merged**